Tests/Sentry.Xamarin.Tests/Sentry.Xamarin.Tests.csproj

```
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+  <TargetFramework>netcoreapp3.1</TargetFramework>
+    <IsPackable>false</IsPackable>
+    <!-- https://github.com/tonerdo/coverlet/issues/33#issuecomment-382748414 -->
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <IsTestProject>true</IsTestProject>
+  </PropertyGroup>
+  <ItemGroup>
+	  <ProjectReference Include="..\..\Src\Sentry.Xamarin\Sentry.Xamarin.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.3" />
+		<PackageReference Include="NSubstitute" Version="4.2.2" />
+		<PackageReference Include="FluentAssertions" Version="5.10.3" />
+		<PackageReference Include="xunit" Version="2.4.1" />
+		<PackageReference Include="xunit.runner.visualstudio" Version="2.4.3" />
+		<PackageReference Include="GitHubActionsTestLogger" Version="1.1.2" />
+	</ItemGroup>		
```

It's misaligned (and mixed tabs and spaces):
```suggestion
  <ItemGroup>
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.3" />
    <PackageReference Include="NSubstitute" Version="4.2.2" />
    <PackageReference Include="FluentAssertions" Version="5.10.3" />
    <PackageReference Include="xunit" Version="2.4.1" />
    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3" />
    <PackageReference Include="GitHubActionsTestLogger" Version="1.1.2" />
  </ItemGroup>		
```